glib/gobject-sys/Cargo.toml

```diff
@@ -98,3 +98,7 @@ version = "2.81"
 rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs", "--generate-link-to-definition"]
 all-features = true
+
+[lints.rust.unexpected_cfgs]
+level = "warn"
+check-cfg = ["cfg(msvc_dll)"]
```

Comment:
This is not necessary anymore fortunately

Reply:
Sorry, my bad, I removed it but I didn't add the changes to the last commit.